third-party/chpl-venv/README.md

```diff
@@ -51,51 +53,134 @@ Used with Sphinx to parse Chapel directives in ReStructured Text generated by
 
 **License**: Apache v2.0
 
-**Website**: https://pypi.python.org/pypi/sphinxcontrib-chapeldomain
+**Website**: https://pypi.org/project/sphinxcontrib-chapeldomain/
 
-### sphinx_rtd_theme
+### sphinx-rtd-theme
 
 ReadTheDocs.org theme for Sphinx
 
 Use with Sphinx as theme for html documentation.
 
 **License**: MIT
 
-**Website**: https://pypi.python.org/pypi/sphinx_rtd_theme
+**Website**: https://pypi.org/project/sphinx-rtd-theme/
+
+### breathe
 
-### argparse
+Sphinx doxygen renderer
 
-Command line argument parsing library, backported from Python 2.7.
+Used with Sphinx to support the compiler developer documentation.
 
-**License**: Python Software Foundation License
+**License**: BSD
 
-**Website**: https://pypi.python.org/pypi/argparse
+**Website**: https://pypi.org/project/breathe/
 
 ### PyYAML
 
-A YAML parser and emitter for Python
+A YAML parser and emitter for Python.
+
+**License**: MIT
+
+**Website**: https://pypi.org/project/PyYAML/
+
+### argcomplete
+
+Tab completion of arguments.
+
+Used by `start_test` to enable tab completion.
+
+**License**: Apache v2.0
+
+**Website**: https://pypi.org/project/argcomplete/
+
+### filelock
+
+A platform independent file lock.
+
+Used by `start_test` to make our execution limiter race free, to avoid
+timeouts while testing.
+
+**License**: Unlicense
+
+**Website**: https://pypi.org/project/filelock/
+
+### setuptools
+
+Easily download, build, install, upgrade, and uninstall Python packages.
+
+Used by `start_test` to install Arkouda's dependencies.
+
+**License**: MIT
+
+**Website**: https://pypi.org/project/setuptools/
+
+### junitparser
+
+Manipulates JUnit/xUnit Result XML files.
+
+Used by `nightly` to parser the Chapel Language Server tests.
+
+**License**: Apache v2.0
+
+**Website**: https://pypi.org/project/junitparser/
+
+### pycparser
+
+C parser in Python.
+
+Required by c2chapel.
+
+**License**: BSD
+
+**Website**: https://pypi.org/project/pycparser/
+
+### pycparserext
+
+Extensions for pycparser.
+
+Required by c2chapel.
 
 **License**: MIT
 
-**Website**: https://pypi.python.org/pypi/PyYAML
+**Website**: https://pypi.org/project/pycparserext/
+
+### pytest
 
-### virtualenv
+A well-known Python testing framework.
 
-Virtual Python Environment builder
+Required by the CLS.
 
-Used to create isolated environment to install other python packages, without
-interfering with system python packages.
+**License**: MIT
+
+**Website**: https://pypi.org/project/pytest/
 
-**License**: Python Software Foundation License
+### pytest-lsp
 
-**Website**: https://pypi.python.org/pypi/virtualenv
+A pytest plugin for end-to-end testing of language servers.
+
+Required by the CLS.
+
+**License**: MIT
+
+**Website**: https://pypi.org/project/pytest-lsp/
 
 Package Dependencies
```

Comment:
There are several other packages required by chapel-py, CLS, and chplcheck that should be listed here